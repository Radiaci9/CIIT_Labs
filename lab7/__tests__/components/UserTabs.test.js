import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import UserTabs from '../../src/components/UserTabs'

describe('<UserTabs />', () => {
  describe('render()', () => {
    test('renders the component', () => {
      const wrapper = shallow(<UserTabs tabs={["tab1"]}/>)

      expect(toJson(wrapper)).toMatchSnapshot()
    })
  })
})
