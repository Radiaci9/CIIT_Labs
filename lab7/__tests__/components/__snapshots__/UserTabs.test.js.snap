// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserTabs /> render() renders the component 1`] = `
<div
  style={
    Object {
      "height": "100%",
    }
  }
>
  <Nav
    tabs={true}
    tag="ul"
    vertical={false}
  >
    <NavItem
      key="0"
      tag="li"
    >
      <NavLink
        active={true}
        onClick={[Function]}
        tag="a"
      >
        tab1
      </NavLink>
    </NavItem>
  </Nav>
  <TabContent
    activeTab={0}
    style={
      Object {
        "height": "100%",
      }
    }
    tag="div"
  >
    <TabPane
      key="0"
      style={
        Object {
          "height": "100%",
        }
      }
      tabId={0}
      tag="div"
    >
      <Row
        style={
          Object {
            "margin": "20px 0 20px 0",
          }
        }
        tag="div"
      >
        <Col
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Label
            check={true}
            className="float-right"
            tag="label"
            widths={
              Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ]
            }
          >
            <Input
              type="checkbox"
            />
             
            Edit
          </Label>
        </Col>
      </Row>
      <Row
        style={
          Object {
            "height": "80%",
          }
        }
        tag="div"
      >
        <Input
          type="textarea"
        />
      </Row>
    </TabPane>
  </TabContent>
</div>
`;
